app/src/main/java/com/kickstarter/viewmodels/BackingAddOnsFragmentViewModel.kt

```diff
@@ -191,27 +192,34 @@ class BackingAddOnsFragmentViewModel {
                     .combineLatest(this.retryButtonPressed.startWith(false), projectAndReward) { _, projectAndReward ->
                         return@combineLatest this.apiClient
                                 .fetchShippingRules(projectAndReward.first, projectAndReward.second)
-                                .doOnError { this.showErrorDialog.onNext(true) }
+                                .doOnError {
+                                    this.showErrorDialog.onNext(true)
+                                    this.shippingSelectorIsGone.onNext(true)
+                    }
                                 .onErrorResumeNext(Observable.empty())
                     }
                     .switchMap { it }
                     .map { it.shippingRules() }
                     .compose(bindToLifecycle())
-                    .subscribe(shippingRules)
+                    .subscribe {
+                        shippingRules.onNext(it)
+                        this.shippingSelectorIsGone.onNext(false)
+                    }
 
             Observable
                     .combineLatest(this.retryButtonPressed.startWith(false), project) { _, pj ->
                         return@combineLatest this.apolloClient
                                 .getProjectAddOns(pj.slug()?.let { it } ?: "")
-                                .doOnError { this.showErrorDialog.onNext(true) }
+                                .doOnError {
+                                    this.showErrorDialog.onNext(true)
+                                    this.shippingSelectorIsGone.onNext(true)}
                                 .onErrorResumeNext(Observable.empty())
                     }
                     .switchMap { it }
                     .filter { ObjectUtils.isNotNull(it) }
-                    .map { listAddOns ->
```

Comment:
This should remain as it was. This is the filter in case is not available the addOn